policy/modules/services/aptcacher.if

```diff
@@ -0,0 +1,21 @@
+## <summary>apt-cacher, cache for Debian APT repositories.</summary>
+
+######################################
+## <summary>
+##	read aptcacher config
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to read it.
+##	</summary>
+## </param>
+#
+interface(`aptcacher_read_config',`
+	gen_require(`
+		type aptcacher_etc_t;
+	')
+
+	files_search_etc($1)
+	allow $1 aptcacher_etc_t:dir list_dir_perms;
+	allow $1 aptcacher_etc_t:file mmap_read_file_perms;
+')
```

Comment:
Is this interface used anywhere?

The summary say's "read aptcacher config" but that is not accurate. I would omit the map here, and create another separate `aptcacher_mmap_read_config_files()` (and maybe even yet another `aptcacher_mmap_config_files() to optionally be used with `aptcacher_read_config()`

Reply:
Drop this, it's not used